src/transformers/tokenization_utils.py

```diff
@@ -1410,3 +1410,147 @@ def clean_up_tokenization(out_string):
             .replace(" 're", "'re")
         )
         return out_string
+
+
+class FastPreTrainedTokenizer(PreTrainedTokenizer):
```

Comment:
We should also have token string and id attributes like `pad_token` and - pad_token_id` that we use often.

We could do complex stuff but maybe a simple way to handle that would be to have a single configuration for models and tokenizers which would store these attributes anyway, as we've started discussing recently with @julien-c

Reply:
Open to other options though @n1t0

Reply:
I'm not entirely sure to understand what you mean.

The attributes are inherited from `PreTrainedTokenizer` so it should be almost the same. The only problem at the moment is when we update these after initialization, it doesn't call `_update_special_tokens()`

Reply:
Oh indeed, my bad